src/runtime/manifest-ast-nodes.ts

```diff
@@ -243,6 +243,7 @@ export interface Particle extends BaseNode {
 export interface ParticleClaimStatement extends BaseNode {
   kind: 'particle-trust-claim';
   handle: string;
+  fieldPath: string[];
```

Comment:
Don't we need to support other kinds of selectors, such as accessing an element in a collection, dereferencing, etc.?

Reply:
My intention was to use dots for all of those selectors too. e.g. `someFieldCollection.someReference.someOtherField` would descend into the collection's inner type, and into the reference's inner type too. 

I'll add appropriate tests for all those cases once I add the code to validate the fields.

Reply:
How would we know what the kind of selector is? Do we infer it from the type essentially?